terraform/groups/provisioning-role/module-iam-policies/terraform-apply.tf

```diff
@@ -0,0 +1,34 @@
+data "aws_iam_policy_document" "apply" {
+  statement {
+    sid       = "ApplyPolicyAllResources"
+    effect    = "Allow"
+    resources = ["*"]
+    actions   = [
+      "ec2:CreateSecurityGroup",
+      "ec2:CreateTags",
+      "lambda:CreateFunction",
+    ]
+  }
+
+  statement {
+    sid       = "ApplyPolicyListedResources"
+    effect    = "Allow"
+    resources = [
+      "arn:aws:iam::${var.aws_account_id}:role/payment-reconciler-execution-*",
+      "arn:aws:events:eu-west-2:${var.aws_account_id}:rule/payment-reconciler-*",
+      "arn:aws:ec2:eu-west-2:${var.aws_account_id}:security-group/*",
+      "arn:aws:lambda:eu-west-2:${var.aws_account_id}:function:payment-reconciler-devops1",
```

Comment:
Environment specific, needs to be changed to a variable

Reply:
I'll close this one and refer to the one below (which mentions all cases of this).